src/createServer.js

```diff
@@ -1,3 +1,59 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const {
+  NO_TEXT_ERROR_MESSAGE,
+  NO_CASE_ERROR_MESSAGE,
+  STATIC_CASES,
+  INVALID_CASE_ERROR_MESSAGE,
+} = require('./constatnts');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url, 'http://localhost:5700');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const textFromUrl = url.pathname.slice(1);
+    const caseFromUrl = url.searchParams.get('toCase');
+
+    const errors = {
+      errors: [],
+    };
+
+    if (textFromUrl === '') {
+      errors.errors.push({ message: NO_TEXT_ERROR_MESSAGE });
+    }
+
+    if (!caseFromUrl) {
+      errors.errors.push({ message: NO_CASE_ERROR_MESSAGE });
+    }
+
+    if (caseFromUrl && !STATIC_CASES.includes(caseFromUrl)) {
+      errors.errors.push({ message: INVALID_CASE_ERROR_MESSAGE });
+    }
+
+    if (!errors.errors.length) {
+      const {
+        originalCase,
+        convertedText,
+      } = convertToCase(textFromUrl, caseFromUrl);
+
+      const result = {
+        originalCase,
+        targetCase: caseFromUrl,
+        originalText: textFromUrl,
+        convertedText,
+      };
+
+      res.end(JSON.stringify(result));
+    } else {
+      res.end(JSON.stringify(errors));
```

Comment:
I cannot find where you add some code to your response headers.
You should respond with status 200 and `OK` status text.
If something is not correct, you should respond with 400 status, `Bad request` statusText and the next payload:
```
{
  "errors": [
    {
      "message": "<SPECIFIC MESSAGE TEXT HERE>"
    }
  ],
}
```